### PR TITLE
Capitalize all UI `h1 title is-3` titles

### PR DESCRIPTION
**Pull request by Monkeychip**
**Notes:** 
* This PR only address `<h1 class="title is-3">Title Here</h1>`. The next PR will address making only the first word caps for all `<h3 class="title is-5">Sub titles</h3>`.
* The only cases I did not capitalize are when we're passing in a user defined text. Such as a secret name, role name, etc.
* This PR also caps the Side navbar headings (confirmed with design)
* We are not capitalizing overview card titles.
* multi-factor authentication is capitalized as follows "Multi-Factor Authentication."
* In another PR, I'll add documentation to our md docs.

### CODE REVIEW

File: ui/app/components/sidebar/nav/access.hbs
Comment (Monkeychip) [inline ui/app/components/sidebar/nav/access.hbs:1]: image

File: ui/app/components/sidebar/nav/cluster.hbs
Comment (Monkeychip) [inline ui/app/components/sidebar/nav/cluster.hbs:1]: image

File: ui/app/templates/components/secret-edit.hbs [hunk 1]
```diff
@@ -12,11 +12,11 @@
     <p.levelLeft>
       <h1 class="title is-3">
         {{#if (eq @mode "create")}}
-          Create secret
+          Create Secret
         {{else if (and this.isV2 (eq @mode "edit"))}}
```
Comment (Monkeychip) [inline ui/app/templates/components/secret-edit.hbs:16]: image

File: ui/app/templates/components/secret-edit.hbs [hunk 2]
```diff
@@ -12,11 +12,11 @@
...
-          Create new version
+          Create New Version
         {{else if (eq @mode "edit")}}
-          Edit secret
+          Edit Secret
```
Comment (Monkeychip) [inline ui/app/templates/components/secret-edit.hbs:19]: I'm not sure you can "edit" a secret. You can only create a new version? Something to be addressed in the new KV work.

File: ui/app/templates/vault/cluster/secrets/backend/diff.hbs
Comment (Monkeychip) [inline ui/app/templates/vault/cluster/secrets/backend/diff.hbs:1]: image

File: ui/app/templates/vault/cluster/secrets/backend/sign.hbs
Comment (Monkeychip) [inline ui/app/templates/vault/cluster/secrets/backend/sign.hbs:1]: image

File: ui/app/templates/vault/cluster/settings/seal.hbs
Comment (Monkeychip) [inline ui/app/templates/vault/cluster/settings/seal.hbs:1]: image

File: ui/app/templates/components/transit-edit.hbs
Comment (Monkeychip) [inline ui/app/templates/components/transit-edit.hbs:1]: image

File: ui/app/templates/vault/cluster/access/mfa/enforcements/index.hbs
[the remark this answers is not in the document]
Comment (Monkeychip) [reply ui/app/templates/vault/cluster/access/mfa/enforcements/index.hbs:1]: Confirmed with design that we want to caps "Factor". Follows AP style guide for compound titles.

File: ui/lib/kmip/addon/templates/role/edit.hbs
Comment (Monkeychip) [inline ui/lib/kmip/addon/templates/role/edit.hbs:1]: image

File: ui/lib/kmip/addon/templates/scopes/create.hbs
Comment (Monkeychip) [inline ui/lib/kmip/addon/templates/scopes/create.hbs:1]: image

File: ui/lib/kubernetes/addon/components/page/configure.hbs
Comment (Monkeychip) [inline ui/lib/kubernetes/addon/components/page/configure.hbs:1]: image

File: ui/lib/kubernetes/addon/components/page/role/create-and-edit.hbs
Comment (Monkeychip) [inline ui/lib/kubernetes/addon/components/page/role/create-and-edit.hbs:1]: image

File: ui/lib/open-api-explorer/addon/templates/components/swagger-ui.hbs
Comment (Monkeychip) [inline ui/lib/open-api-explorer/addon/templates/components/swagger-ui.hbs:1]: image

File: ui/lib/pki/addon/components/page/pki-tidy-auto-configure.hbs
Comment (Monkeychip) [inline ui/lib/pki/addon/components/page/pki-tidy-auto-configure.hbs:1]: image

File: ui/lib/pki/addon/components/page/pki-tidy-manual.hbs
Comment (Monkeychip) [inline ui/lib/pki/addon/components/page/pki-tidy-manual.hbs:1]: image

File: ui/lib/pki/addon/templates/certificates/certificate/details.hbs
Comment (Monkeychip) [inline ui/lib/pki/addon/templates/certificates/certificate/details.hbs:1]: image

File: ui/lib/pki/addon/templates/configuration/edit.hbs
Comment (Monkeychip) [inline ui/lib/pki/addon/templates/configuration/edit.hbs:1]: image

**Review by hellobontempo — APPROVED** [review]

**Review by hellobontempo — COMMENTED** [review]
Could this part of the description clarify that we'll use sentence case for overview cards?

> We are not capitalizing overview card titles.

**Pull request merged**